TAT.homework5_2/Main.java

```diff
@@ -0,0 +1,43 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Defines an entry point for application providing creation of html-table
+ * consisting list of servers with their response time
+ * @author Yury Suponev
+ */
+public class Main {
+  /**
+   * Files: to read (containing servers list), to write (table will be here)
+   */
+  private static final String FILE_TO_READ = "server_list.txt";
+  private static final String FILE_TO_WRITE = "servers_table.html";
+
+  /**
+   * Creates html-file containing table with servers IPs and their pings
+   * Reads servers IPs from command line if it isn't empty, otherwise reads from file
+   * @param args command line parameters
+   */
+  public static void main(String[] args) {
+    ArrayList<String> ipList;
+    IpReader ipReader = new IpReader();

```

Comment:
If you split classes as I suggested it'll be as easy as IpReader ipReader = (args.length != 0) ? new CommandLineArgsIpReader(args) : new FileIpReader(FILE_TO_READ);